### PR TITLE
[Mobile app] resolving flutter linter issues.

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Marks some linter rules as ignored.

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state
- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

### CODE REVIEW

File: mobile/lib/screens/map/map_view_widgets.dart [hunk 1]
```diff
@@ -158,47 +158,44 @@ class SearchTile extends StatelessWidget {
...
-
-      // TODO: Substitute with widget
-      if (nearestSite == null) {
+    )
+        .then((place) async {
+      if (place != null) {
+        await LocationService.getNearestSite(
+          place.latitude,
+          place.longitude,
+        ).then((nearestSite) async {
```
Comment (NoahNsimbe) [inline mobile/lib/screens/map/map_view_widgets.dart:170]: The map search functionality should work

File: mobile/lib/widgets/error_widgets.dart [hunk 2]
```diff
@@ -559,8 +559,8 @@ class AppCrushWidget extends StatelessWidget {
             const Spacer(),
             InkWell(
               onTap: () async {
+                // TODO log to  a backend service
                 PackageInfo packageInfo = await PackageInfo.fromPlatform();
-                String email = "support@airqo.net";
```
Comment (Baalmart) [inline mobile/lib/widgets/error_widgets.dart:563]: Yeah, thanks @NoahNsimbe for this removal.
I will set up a special notification tool for all of us, soon. 
Let us first get done with Access Control.

File: mobile/lib/screens/web_view_page.dart [hunk 3]
```diff
@@ -4,7 +4,9 @@ import 'package:app/widgets/widgets.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:webview_flutter/webview_flutter.dart';
+// ignore: depend_on_referenced_packages, required by package
```
Comment (Baalmart) [inline mobile/lib/screens/web_view_page.dart:7]: not sure if we need these comments, do we? They do not seem to be important? Thoughts? @NoahNsimbe

File: mobile/lib/screens/map/map_view_widgets.dart (hunk 1 above)
Comment (Baalmart) [reply mobile/lib/screens/map/map_view_widgets.dart:170]: thanks Noah!

**Review by Baalmart — APPROVED** [review]
Thanks @NoahNsimbe , I have left a few comments for your consideration later.

File: mobile/lib/screens/web_view_page.dart (hunk 3 above)
Comment (NoahNsimbe) [reply mobile/lib/screens/web_view_page.dart:7]: These comments prevent dart code metrics from raising errors like [depend_on_referenced_packages](https://dart-lang.github.io/linter/lints/depend_on_referenced_packages.html)

**Pull request merged**